android/src/main/java/com/oblador/keychain/cipherStorage/CipherStorageBase.java

```diff
@@ -505,7 +504,7 @@ public static String getDefaultAliasIfEmpty(@Nullable final String service, @Non
     public static IvParameterSpec readIv(@NonNull final byte[] bytes) throws IOException {
       final byte[] iv = new byte[IV_LENGTH];
 
-      if (IV_LENGTH <= bytes.length)
```

Comment:
When switching over to this overloaded method handling `byte[]`, this condition was triggered. Reading through this method, I believe the intent may have been to validate that the input bytes array is actually *greater* than `IV_LENGTH`.

Reply:
Whoa, strange bug. Wonder if this code was ever used before.